src/SFA.DAS.AANHub.Domain/Entities/Admin.cs

```diff
@@ -5,6 +5,7 @@ public class Admin
     {
         public Guid MemberId { get; set; }
         public string? Email { get; set; }
+        public string? UserName { get; set; }
         public DateTime? LastUpdated { get; set; }
```

Comment:
LastUpdated is now NOT NULL in src/SFA.DAS.AANHub.Database/Tables/Admin.sql. Perhaps this could mean LastUpdated need not be nullable DateTime (compared with Created in EntityBase.cs)